systemtest/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+# get used jpscore executable
+get_property(jpscore_name TARGET jpscore PROPERTY NAME)
```

Comment:
Maybe also check if `OUTPUT_NAME` is empty, that could be useful if we set the name by hand.

Reply:
Changed to generator expression